packages/support/lib/fs.js

```diff
@@ -32,9 +35,9 @@ const fs = {
   exists (path) { return fs.hasAccess(path); },
   rimraf: B.promisify(rimraf),
   rimrafSync: rimraf.sync.bind(rimraf),
-  async mkdir (...args) {
+  async mkdir (filepath, opts = {}) {
```

Comment:
shall we check other org modules for possible usages of special args?

Reply:
much like `ncp` above, `mkdir` only accepts three params (`dirname`, `opts`, and a `callback`).  so we only need the first two.